themes/beautifulhugo/layouts/partials/script.html

```diff
@@ -0,0 +1,75 @@
+<!-- # custom 'script' partial -->
+<script>
+//hide code blocks labeled w/ "language-test"
+$('.language-test').parent().css('display', 'none')
+
+// Add a Copy button for each example code snippet:
+var copybtns = {
+  fallbackCopyTextToClipboard: function (text) {
+    var textArea = document.createElement("textarea");
+    textArea.value = text;
+    
+    // Avoid scrolling to bottom
+    textArea.style.top = "0";
+    textArea.style.left = "0";
+    textArea.style.position = "fixed";
+  
+    document.body.appendChild(textArea);
+    textArea.focus();
+    textArea.select();
+  
+    try {
+      var successful = document.execCommand('copy');
+      var msg = successful ? 'successful' : 'unsuccessful';
+      console.log('Fallback: Copying text command was ' + msg);
```

Comment:
No need for console.log() in the final version except maybe for the catch block.